### DISCUSSION TITLE
Anyone have issues with Discord RPC being stuck?

### DISCUSSION THREAD

**Discussion by kinoko87** [KadeArchive/Kade-Engine#1516] [Q&A]
I'm modding a new week in Kade Engine but now my discord status is always stuck saying I'm on Week 7 in FNF, even if I'm not coding, playing Kade Engine/FNF, hell even when I only have discord open!

I am unfortunately using an older version of Kade Engine since the programming I did broke on the latest update for no reason and even when I reverted to old code, it still failed to build properly.

The KE Version my team is using is 1.5.4. I have tried removing all the RPC code or just having DiscordClient.shutdown() only in the code.

**Comment by Riskerman21** [comment]
hey discord rpc is buggy always you can have shot recoding it. Complain about in there repository(KE has nothing with this):
https://github.com/discord/discord-rpc
